db.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+$connect = mysqli_connect("localhost", "root", "root", "readme");
+if ($connect == false) {
+    print("Ошибка подключения: " . mysqli_connect_error());
+} else {
+    $query = "SELECT name, icon_class from content_types";
```

Comment:
Для глаза слишком много ветвлений if else.
Как хорошую практику, советую максимально уходить от else. 

Здесь это можно решить таким способом:

```suggestion
if ($connect === false) {
    die("Ошибка подключения: " . mysqli_connect_error());
}
$query = "SELECT name, icon_class from content_types";
// и так далее
```